tests/system/test_aggregation_query.py

```diff
@@ -70,54 +70,156 @@ def nested_query(aggregation_query_client, ancestor_key):
 
```

Comment:
I added some tests to make sure the transaction id is captured when running a query inside the context of a transaction

Let me know if you had anything else in mind. I'm still trying to get my head around the relationship between transactions and queries, and I'm not seeing any existing tests or samples around this

Reply:
Looks like there are a few examples here: https://github.com/googleapis/python-datastore/blob/main/tests/system/test_transaction.py, if you want to move them into there instead (I don't have a strong preference)

Reply:
Yeah I was a little unsure which file to put these in since they involves both queries and transactions. I just looked into moving these to test_transaction.py, but all the test fixtures for configuring queries live in this file, so it's probably easiest to keep it here